scripts/test_data/ny.avsc

```diff
@@ -0,0 +1,82 @@
+{
```

Comment:
lets see if we re-use the stocks data we have. 
data is located at HUID_REPO/docker/demo/data/batch_1.json